src/models/RoomMessagesResponse.ts

```diff
@@ -0,0 +1,12 @@
+import { RoomEventData, RoomStateEventData } from "./events/RoomEvent";
+
+/**
+ * The options available when creating a room.
```

Comment:
Not the right docs. However, given the translation we'll need to do on the returned events anyways this interface can be deleted.